### ISSUE TITLE
MODX 3 setup: each() function is deprecated

### ISSUE THREAD

**Issue by pbowyer** [modxcms/revolution#13848]
### Summary
Deprecation notice when installing MODX3

### Step to reproduce
Set up a server using PHP 7.2
Follow the instructions on https://modx3.org to install from Git

Go to http://example.org/setup and follow through

### Observed behavior
On the final "Installation Summary" screen I get:
```

Deprecated: The each() function is deprecated. This message will be suppressed on further calls in /var/www/web/setup/includes/runner/modinstallrunner.class.php on line 185
--


1 | 0.0000 | 399000 | {main}( ) | .../index.php:0
2 | 0.0055 | 573944 | modInstallRequest->handle( ) | .../index.php:103
3 | 0.0085 | 662312 | include( '/var/www/web/setup/controllers/install.php' ) | .../modinstallrequest.class.php:78
4 | 0.0144 | 671960 | modInstallRunnerWeb->run( ) | .../install.php:19
5 | 0.0145 | 671992 | modInstallRunnerWeb->execute( ) | .../modinstallrunner.class.php:39
6 | 0.0145 | 671992 | modInstallRunnerWeb->writeConfig( ) | .../modinstallrunnerweb.class.php:23
7 | 0.0308 | 679120 | each ( ) | .../modinstallrunner.class.php:185
```

### Expected behavior
No deprecation warning

### Environment
MODX 3.0.0-dev (commit 614bdc1a4b5cb8ff4e41e2432d7ff059ecc23557)
Apache
PHP 7.2 as FPM

**Comment by alroniks** [comment]
It was fixed in 2.6.2, but 2.6.x still not merged into 3.x. I guess it will be solved a bit later during one big merge.

**Comment by alroniks** [comment]
As 2.6.x was merged into 3.x some weeks before and search by each() in 3.x codebase doesn't get results it seems solved. Closing it.
